src/styles/index.scss

```diff
@@ -2,3 +2,7 @@
 @import '@fontsource/roboto/400.css';
 @import '@fontsource/roboto/500.css';
 @import '@fontsource/roboto/700.css';
+
+body {
+    margin: 0px;
```

Comment:
I found the component `CssBaseline`, which does the same thing and even more. You can use it in `index.tsx`